doc/guide/build.hpp

```diff
@@ -249,11 +250,37 @@ suite.  You can build this component with
 $ make mlpack_test
 @endcode
 
-and then run all of the tests, or an individual test suite:
+We use Catch2 to write our tests
```

Comment:
```suggestion
We use <a href="https://github.com/catchorg/Catch2">Catch2</a> to write our tests.
```
Might be nice to include a URL too. :)

Reply:
@rcurtin Yes sure ..... I think i will be good to use this same URL for Catch2 here in test docs readme.md(https://github.com/mlpack/mlpack/blob/master/src/mlpack/tests/README.md) as the existing one is showing 404 error.

Reply:
Sounds good---do you want to include the link here in `build.hpp` too?